tests/unit_tests/test_server_client.py

```diff
@@ -1725,6 +1725,90 @@ async def test_feedback_succeeds_when_langsmith_enabled() -> None:
                 assert json_response == expected_response_json
 
 
+async def test_feedback_defaults_on_for_hosted() -> None:
+    """Tests that the feedback endpoint can accept feedback to langsmith."""
+
+    with patch("langserve.api_handler.ls_client") as mocked_ls_client_package:
+        with patch("langserve.api_handler.tracing_is_enabled") as tracing_is_enabled:
+            tracing_is_enabled.return_value = True
+            mocked_client = MagicMock(return_value=None)
+            mocked_ls_client_package.Client.return_value = mocked_client
+            mocked_client.create_feedback.return_value = ls_schemas.Feedback(
+                id="5484c6b3-5a1a-4a87-b2c7-2e39e7a7e4ac",
+                created_at=datetime.datetime(1994, 9, 19, 9, 19),
+                modified_at=datetime.datetime(1994, 9, 19, 9, 19),
+                run_id="f47ac10b-58cc-4372-a567-0e02b2c3d479",
+                key="silliness",
+                score=1000,
+            )
+
+            local_app = FastAPI()
+
+            # This is the hackiest code ever, but here's how it goes:
```

Comment:
wishful thinking from copilot...

<img width="545" alt="Screenshot 2023-12-07 at 8 53 58 PM" src="https://github.com/langchain-ai/langserve/assets/9028897/a75c53fc-05c0-47cf-9482-c754c1bdb828">

Reply:
it's fine :)